tests_e2e/pipeline/pipeline.yml

```diff
@@ -46,7 +46,7 @@ parameters:
   - name: collect_lisa_logs
     displayName: Collect LISA logs
     type: boolean
-    default: true
```

Comment:
i've been meaning to disable this for several PRs; finally, I remember to do it in this one.